Minkowski_sum_2/test/Minkowski_sum_2/CMakeLists.txt

```diff
@@ -6,6 +6,16 @@ project( Minkowski_sum_2_Tests )
 
 cmake_minimum_required(VERSION 2.8.10)
 
+list(FIND CMAKE_CXX_COMPILE_FEATURES cxx_generalized_initializers has_cpp11)
+if (has_cpp11 LESS 0)
+  message(STATUS "NOTICE: These examples requires a C++11 compiler and will not be compiled.")
+  return()
+endif()
+
+# Use C++11 for this directory and its sub-directories.
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD_REQUIRED TRUE)
+
```

Comment:
Efi, would you mind fixing that? It was already merged in `master`, but I think you could add that "fix" in the PR about Arr_2: https://github.com/CGAL/cgal/pull/3093